packages/hub/src/scope.ts

```diff
@@ -236,6 +238,52 @@ export class Scope implements ScopeInterface {
     return newScope;
   }
 
+  /**
+   * @inheritDoc
+   */
```

Comment:
We can't inherit this from anywhere since it's not on the interface ^^
Please explain what it does